aeppl/joint_logprob.py

```diff
@@ -47,11 +46,11 @@ def factorized_joint_logprob(
         Y \sim& \operatorname{N}(0, \sigma^2)
 
     If we create a value variable for ``Y_rv``, i.e. ``y = at.scalar("y")``,
-    the graph of ``joint_logprob({Y_rv: y})`` is equivalent to the
-    conditional probability :math:`\log p(Y = y \mid \sigma^2)`.  If we specify
-    a value variable for ``sigma2_rv``, i.e. ``s = at.scalar("s2")``, then
-    ``joint_logprob({Y_rv: y, sigma2_rv: s})`` yields the joint
-    log-probability
+    the graph of ``factorized_joint_logprob({Y_rv: y})`` is equivalent to the
+    conditional probability :math:`\log p(Y = y \mid \sigma^2)`, with a stochastic
+    ``sigma2_rv``. If we specify a value variable for ``sigma2_rv``, i.e.
+    ``s = at.scalar("s2")``, then ``factorized_joint_logprob({Y_rv: y, sigma2_rv: s})``
+    yields the joint log-probability of the two variables.
```

Comment:
```suggestion
    the graph of ``factorized_joint_logprob({Y_rv: y_vv})`` is equivalent to the
    conditional probability :math:`\log p(Y = y \mid \sigma^2)`, with a stochastic
    ``sigma2_rv``. If we specify a value variable for ``sigma2_rv``, i.e.
    ``s_vv = at.scalar("s2")``, then ``factorized_joint_logprob({Y_rv: y_vv, sigma2_rv: s_vv})``
    yields the joint log-probability of the two variables.
```

This only adds the `*_vv` suffix convention for value variables.